jobs/ServiceInstanceUpdateJob.js

```diff
@@ -72,9 +73,18 @@ class ServiceInstanceUpdateJob extends BaseJob {
 
   static updateInstanceIfOutdated(instanceDetails, planId, operationResponse) {
     const plan = catalog.getPlan(planId);
+    let context;
     logger.info(`Instance Id: ${instanceDetails.instance_id} - manager : ${plan.manager.name} - Force Update: ${plan.service.force_update}`);
-    return this
-      .getOutdatedDiff(instanceDetails.deployment_name, instanceDetails.instance_id, plan)
+    return cloudController.getOrgAndSpaceGuid(instanceDetails.instance_id)
```

Comment:
why is this CF call needed? If we need to fetch the org and space, fetch it from apiserver resource.